include/impl/process.inl

```diff
@@ -149,14 +149,8 @@ namespace ntw::obj {
             auto      attributes = make_attributes(nullptr, 0);
             CLIENT_ID cid{ reinterpret_cast<void*>(pid), nullptr };
 
-            void*      temp_handle = nullptr;
-            const auto status =
-                LI_NT(NtOpenProcess)(&temp_handle, access, &attributes, &cid);
-
-            if(NT_SUCCESS(status))
-                _handle.reset(temp_handle);
-
-            return status;
+            _handle.reset();
```

Comment:
The old version retained the old handle if opening new one failed. While this is not related to exceptions I like to look at it as if it were and with this change we are loosing the strong exception safety guarantee